### PR TITLE
Adds admin freezing

### PR DESCRIPTION
**Pull request by alexkar598**
:cl: alexkar598
add: Admin freezing is now a thing.FREEZE IN THE NAME OF THE LAW
/:cl:

@steamp0rt im sorry

### CODE REVIEW

File: code/modules/mob/mob_defines.dm [hunk 1]
```diff
@@ -110,3 +110,8 @@
 	var/datum/click_intercept
 
 	var/registered_z
+//sunset start
+	var/adminfrozen = 0 //handle for someone's sleeping time from before they got frozen
```
Comment (ghost) [inline code/modules/mob/mob_defines.dm]: You can just move this to `sunset/code/modules/mob/mob_defines.dm `

Comment (ghost) [inline code/modules/mob/mob_defines.dm]: `FALSE`, not `0`

File: sunset/code/modules/admin/verbs/randomverbs.dm [hunk 2]
```diff
@@ -0,0 +1,33 @@
+/obj/effect/overlay/adminfrozen
+	name = "FROZEN"
+	icon = 'sunset/icons/effects/effects.dmi'
+	icon_state = "adminfreeze"
+/
```
Comment (ghost) [inline sunset/code/modules/admin/verbs/randomverbs.dm]: What is with the random slash?

**Review by ghost — CHANGES_REQUESTED** [review]

File: sunset/code/modules/admin/verbs/randomverbs.dm (hunk 2 above)
Comment (alexkar598) [reply sunset/code/modules/admin/verbs/randomverbs.dm]: it is the result of 3.5 million years of evolution 


and a typo

File: code/modules/mob/mob_defines.dm (hunk 1 above)
Comment (alexkar598) [reply code/modules/mob/mob_defines.dm]: not a boolean,its the amount of time they were sleeping before the freeze

Comment (ghost) [reply code/modules/mob/mob_defines.dm]: Oh.

File: sunset/code/modules/admin/verbs/randomverbs.dm (hunk 2 above)
Comment (alexkar598) [reply sunset/code/modules/admin/verbs/randomverbs.dm]: fixed

File: code/modules/mob/mob_defines.dm (hunk 1 above)
Comment (alexkar598) [reply code/modules/mob/mob_defines.dm]: done

**Comment by alexkar598** [comment]
@steamp0rt ready for merge,tested and working

**Review by ghost — APPROVED** [review]
LGTM, waiting for Bksparkie to say yes

**Pull request merged**